docs/content/markdown.md

```diff
@@ -62,10 +64,10 @@ The following fields are defined for Bartholomew:
 
 After the `---`, you can write the content's body in Markdown.
 
-## Writing a Markdown Body
+## Writing a markdown body
 
 All documents in Bartholomew are written in Markdown.
-[Markdown]() is a simple text format designed to make it easy (and very fast)
+[Markdown](https://www.markdownguide.org/) is a simple text format designed to make it easy (and very fast)
 to write text that can then be converted into HTML.
 
 For example, the Markdown `*hello*` is transformed into *hello*. And the Markdown
```

Comment:
Note, I'm seeing hello as italicized (_hello_) rather than made bold (**hello**).

I'm assuming we want to update this to `**hello**`

Reply:
There's another instance of this below, as well... should be `**goodbye**`